NuGet/Xamarin.Microsoft.Band.Native.nuspec

```diff
@@ -41,10 +38,13 @@
         <file src="Component/License.md" target="License.md" />
 		
         <file src="Component/Details.md" target="lib\netcore451\Readme.md" />
+	<file src="NuGet/_._" target="lib\netcore451\_._" />
         <file src="Component/Details.md" target="lib\portable-net45+win+wpa81\Readme.md" />
+	<file src="NuGet/_._" target="lib\portable-net45+win+wpa81\_._" />
         <file src="Component/Details.md" target="lib\portable-win81+wpa81\Readme.md" />
+	<file src="NuGet/_._" target="lib\portable-win81+wpa81\_._" />
         <file src="Component/Details.md" target="lib\uap10.0\Readme.md" />
-        <file src="Component/Details.md" target="lib\wpa81\Readme.md" />
+    <file src="NuGet/_._" target="lib\uap10.0\_._" />
```

Comment:
Is there a reason for _._? Doesn't the readme.md perform the same "placeholder" function?